src/plugins/projectexplorer/projectvcsstatus.cpp

```diff
@@ -146,8 +153,10 @@ Core::VcsChangeType ProjectVcsStatus::vcsStatusChanges(ProjectExplorer::Node *no
     }
     else if (auto *folderNode = node->asFolderNode()) {
         if (!changeSet.isFolderNodesCurrent) {
-            changeSet.folderNodes = extractFolderNodes(changeSet.fileChanges, projectPath);
-            changeSet.isFolderNodesCurrent = true;
+            if (auto *project = folderNode->getProject()) {
```

Comment:
Is this really necessary? Is the project we extracted in line 138 not good enough?

Reply:
true